app/util/conf/base_config_loader.py

```diff
@@ -87,6 +87,11 @@ def configure_defaults(self, conf):
         conf.set('git_askpass_exe', join(bin_dir, 'git_askpass.sh'))
         conf.set('git_ssh_exe', join(bin_dir, 'git_ssh.sh'))
 
+        # How slaves get the project
+        # Slaves would get the project from master if set to True. Otherwise it would just get the project in
+        # the same way how the master gets the project.
+        conf.set('get_project_from_master', True)
```

Comment:
Lu and I discussed offline and decided to add the conf setting without changing the default behavior, and postpone changing the default for a future change.